src/bundle/Core/Command/ExpireUserPasswordsCommand.php

```diff
@@ -0,0 +1,383 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\Command;
+
+use Exception;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\ContentList;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalAnd;
+use eZ\Publish\API\Repository\Values\ContentType\ContentType;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use eZ\Publish\API\Repository\Values\Filter\Filter;
+use eZ\Publish\SPI\Persistence\User\Handler;
+use InvalidArgumentException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExpireUserPasswordsCommand extends Command
+{
+    protected static $defaultName = 'ibexa:user:expire-password';
+
+    public const DEFAULT_BATCH_SIZE = 50;
+
+    public const DEFAULT_PASSWORD_TTL = 00;
+
+    public const BEFORE_RUNNING_HINTS = <<<EOT
+<error>Before you continue:</error>
+- Make sure to back up your database.
+- Take installation offline, during the script execution the database should not be modified.
+- Run this command without memory limit.
+- Run this command in production environment using <info>--env=prod</info>
+EOT;
+
+    /** @var \eZ\Publish\API\Repository\Repository */
+    private $repository;
+
+    /** @var \eZ\Publish\SPI\Persistence\User\Handler */
+    private $userHandler;
+
+    public function __construct(
+        Repository $repository,
+        Handler $userHandler
+    ) {
+        $this->repository = $repository;
+        $this->userHandler = $userHandler;
+
+        parent::__construct();
+    }
+
+    protected function configure()
```

Comment:
You can safely add here
```suggestion
    protected function configure(): void
```